includes/class-wc-payments.php

```diff
@@ -1268,6 +1268,90 @@ function wcpay_show_old_woocommerce_for_norway_notice() {
 
 			add_filter( 'admin_notices', 'wcpay_show_old_woocommerce_for_norway_notice' );
 		}
+
+		if ( defined( 'WC_VERSION' ) && version_compare( WC_VERSION, '7.8', '<' ) && get_woocommerce_currency() === 'SEK' ) {
```

Comment:
I'm sure you already thought about it, but can we somehow generalize this conditional notice for all 3 countries?

Reply:
Oh yeah, I tried to move this to a function but there will be too many arguments and it's not readable. I decided to go with duplication to keep things readable but happy to take another path here

Reply:
I think we can refactor the code: 
- create only 1 method for all currencies
```
function wcpay_show_old_woocommerce_notice_for_huf_sek...() {
    $currencies = ['HUF', 'SEK', ...]
     $store_currency = get_woocommerce_currency() ;
    if ( ! defined( 'WC_VERSION' ) || ! version_compare( WC_VERSION, '7.8', '<' ) || ! in_array( $currencies, $store_currency ) ) {
         return;
    }
    switch (store_currency) {
        case 'SEK':
            $notice = The WooCommerce version you have installed is not compatible with WooCommerce Payments for a Swedish business. '
                     break;
            ......
   } 
   ?>
    <div class="notice wcpay-notice notice-error">
      .....
   <?php
}
```
- move this method outside `add_woo_admin_notes` method
- in `add_woo_admin_notes` add `add_filter( 'admin_notices', [ $this, 'wcpay_show_old_woocommerce_notice_for_huf_sek...'] );`

Reply:
I second what @dpaun1985 proposed. There is no point in having three identical methods with different content.

Reply:
Thank you @dpaun1985 and @mordeth. I used your suggestion with a small twist for this 2c49ddfca

It's important to have a string in `__()` (not a variable) so it can be picked for translation.